site/public/js/drag-and-drop.js

```diff
@@ -348,20 +348,6 @@ function handleSubmission(submit_url, return_url, days_late, late_days_allowed,
         // Files selected
         for (var i = 0; i < file_array.length; i++) {
             for (var j = 0; j < file_array[i].length; j++) {
-                if (file_array[i][j].name.indexOf("'") != -1 ||
-		    file_array[i][j].name.indexOf("\"") != -1) {
-                    alert("ERROR! You may not use quotes in your filename: " + file_array[i][j].name);
-                    return;
-                }
-                if (file_array[i][j].name.indexOf("\\") != -1) {
-                    alert("ERROR! You may not use backslash in your filename: " + file_array[i][j].name);
-                    return;
-                }
-                if (file_array[i][j].name.indexOf("<") != -1 ||
-		    file_array[i][j].name.indexOf(">") != -1) {
-                    alert("ERROR! You may not use angle brackets in your filename: " + file_array[i][j].name);
-                    return;
-                }
```

Comment:
Leave this in. It's better to catch this on the front-end as once they submit, there's no way to easily recover what files they had tried to submit and this could be annoying if their upload consists of a bunch of files. We do want to leave the checks in place in the backend as well (so have two sets of checks).